src/SampleIntegration/TracedHog.cs

```diff
@@ -3,8 +3,9 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using System.Text;
-
+using Dynamo.Events;
 using DynamoServices;
+using Newtonsoft.Json;
```

Comment:
Where's newtonsoft package reference in the csproj?

Reply:
DynamoVisualProgramming.Core has a dependency on NewtonSoft.
So it is an indirect reference